lib/tasks/backend.rake

```diff
@@ -20,7 +20,7 @@ Parameters:
 END
   task :fake => :environment do
     cinstance = ENV.fetch('CINSTANCE_ID') do
-      Account.find(ENV.fetch('PROVIDER_ID')).provided_cinstances.live.pluck(:id)
+      Account.find(2).provided_cinstances.live.pluck(:id)
```

Comment:
I think you should revert this, or at least use that `2` as default but still honor `PROVIDER_ID` when provided.

Reply:
Ooops :sweat_smile: 

100%, that just slipped from some local testing I did, and wasn't intended.
Thanks for catching!